devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -101,66 +100,64 @@ def on() {
 	zigbee.on()
 }
 
+def setLevel(value, rate=null) {
+	zigbee.setLevel(value)
+}
+
+def configure() {
+	zigbee.configureReporting(0x0406, 0x0000, 0x18, 30, 600, null) +
+		zigbee.onOffConfig() +
+		zigbee.levelConfig() +
+		refresh()
+}
+
+def updated() {
+	refresh()
+}
+
 def ping() {
-	return zigbee.onOffRefresh()
+	return zigbee.levelRefresh()
 }
 
 def refresh() {
-	zigbee.onOffRefresh() +
+	zigbee.readAttribute(0x0406, 0x0000) +
+		zigbee.onOffRefresh() +
 		zigbee.levelRefresh() +
-		zigbee.colorTemperatureRefresh() +
-		zigbee.onOffConfig(0, 300) +
-		zigbee.levelConfig()
-}
-
-def configure() {
```

Comment:
Why did you remove `configure()` function?

Reply:
Oh, I didn't remove that. I just cleaned up configure() and refresh() as you mentioned before. I separated refresh and configure commands.

Reply:
Oh yeah, you're right. Sorry, missed that.